tests/functionality/testversion.cpp

```diff
@@ -0,0 +1,19 @@
+#include <string>
+#include <iostream>
+
+
+#include <resolve/utilities/version.hpp>
+#include <resolve/utilities/version.cpp>
+
+//author: RD
+//version test to check to make sure ReSolve's version can be printed
+
+
+int main(int argc, char *argv[])
+{
+  std::string versionstr;
+  ReSolveVersionGetVersionStr(versionstr);
+  std::cout<<"ReSolveVersionGetVersionStr Test: "<<versionstr<<std::endl<<std::endl;;
+  
+  return 0;
+}
```

Comment:
I would make this test verify the version, as well.
```suggestion
  std::string versionstr;
  std::string answer("1.0.0-alpha1");
  ReSolveVersionGetVersionStr(versionstr);
  std::cout<<"ReSolveVersionGetVersionStr Test: "<<versionstr<<std::endl<<std::endl;
  
  if (versionstr != answer) {
    std::cout << "Incorrect ReSolve version returned!\n";
    return 1;
  }
  return 0;
}

```
This will force future developers to check if the correct version is set ahead of the release.